src/containers/ViewerPanel/index.tsx

```diff
@@ -6,12 +6,14 @@ import SimulariumViewer, {
     SelectionStateInfo,
 } from "@aics/simularium-viewer";
 import "@aics/simularium-viewer/style/style.css";
-import { TrajectoryFileInfo } from "@aics/simularium-viewer/type-declarations/simularium";
+import {
+    TrajectoryFileInfo,
+    TrajectoryFileInfoV2,
+} from "@aics/simularium-viewer/type-declarations/simularium";
```

Comment:
I would suggest making `TrajectoryFileInfo` always represent the latest version so that you don't need to import two different types here.  Maybe for another PR?

Reply:
See https://github.com/allen-cell-animated/simularium-website/pull/115#discussion_r609134135